### PR TITLE
Add Apache Spark version 3.2.1

### PR DESCRIPTION
**Pull request by gengliangwang**
# Pull request motivation(s)

As Apache Spark 3.2.1 is coming, this PR is to add Apache Spark version 3.2.1

### CODE REVIEW

**Comment by gengliangwang** [comment]
cc @shortcuts @huaxingao @HyukjinKwon

File: configs/apache_spark.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     {
       "url": "https://spark.apache.org/docs/(?P<version>.*?)/",
       "variables": {
-        "version": ["latest", "3.0.3",  "3.1.1", "3.1.2", "3.2.0"]
+        "version": ["latest", "3.0.3", "3.1.1", "3.1.2", "3.2.0", "3.2.1"]
```
Comment (HyukjinKwon) [inline configs/apache_spark.json:7]: I think we remove 3.2.0(?)

Comment (gengliangwang) [reply configs/apache_spark.json:7]: @HyukjinKwon I don't understand. We should keep it.

Comment (HyukjinKwon) [reply configs/apache_spark.json:7]: Oops. sorry I misunderstood. LGTM.

**Review by HyukjinKwon — APPROVED** [review]
LGTM

**Comment by gengliangwang** [comment]
ping @shortcuts again, sorry to bother but we are going the announce the release:
https://github.com/apache/spark-website/pull/375

**Comment by shortcuts** [comment]
Hey, all the DocSearch indices have been migrated to our new infra: https://docsearch.algolia.com/docs/migrating-from-legacy/

You can now edit your config via a web editor and start crawls directly! Please check your emails :D

**Comment by gengliangwang** [comment]
@shortcuts I have edited the configurations. It works in the search preview **but there is no search result in** https://spark.apache.org/docs/3.2.1/ yet
image

Any suggestions? Does the new index take time to become effective? Or we should update the code in Spark?

**Comment by shortcuts** [comment]
Hey @gengliangwang, you also need to change the credentials in your DocSearch instance on the apache spark side. 

apiKey: d62f962a82bc9abb53471cb7b89da35e
appId: RAI69RXRSK

**Comment by gengliangwang** [comment]
@shortcuts Thank you!

**Pull request closed without merging**